### PR TITLE
Update rules

### PR DESCRIPTION
**Pull request by residal**
Updated rules

### CODE REVIEW

File: src/pages/ServerRules.js [hunk 1]
```diff
@@ -54,11 +53,20 @@ class ServerRules extends Component {
 								soliciting, and/or buying anything through our services is not
 								allowed.
 							</li>
-							<li>Discrimination of any kind will not be tolerated.</li>
+							<li>
+								Discrimination of any kind will not be tolerated. This includes discrimination of age,
```
Comment (AtditC) [inline src/pages/ServerRules.js:57]: I disagree with "discrimination of age". If Player A calls Player B "shut up 8 year old", does this mean we should punish him for 2 weeks for discrimination? Discrimination of age makes no sense imo. I suggest removing that part.

Comment (residal) [reply src/pages/ServerRules.js:57]: Well, that situation would likely be considered disrespect or toxicity. Discrimination of age would mean that someone is being treated less favorably based on their age, while ageism is not a top concern on Warzone I do think that some people may dislike being criticized, excluded, or told that they aren't good enough, purely based on their age.

I think discrimination of age can typically be considered as toxicity at most, however we can make a change to it if a Senior Moderator or an Administrator shares their perspective.

File: src/pages/ServerRules.js [hunk 2]
```diff
@@ -54,11 +53,20 @@ class ServerRules extends Component {
...
+								Discrimination of any kind will not be tolerated. This includes discrimination of age, 
+								gender, race, disability, religion, sexual orientation, and more.
+							</li>
 							<li>
 								The use of excessive capitalization, flooding chat with
 								characters, and spamming are all punishable offenses.
 							</li>
+							<li>
+								Evading a punishment will result in a harsher punishment. You are not permitted to speak in #server-chat on 
+								our Discord if you are muted in-game, and you may not join our Discord or Minecraft server with an alternet account
```
Comment (jellz) [inline src/pages/ServerRules.js:66]: The correct spelling is `alternate`

File: src/pages/ServerRules.js (hunk 1 above)
Comment (AtditC) [reply src/pages/ServerRules.js:57]: "Discrimination of age would mean that someone is being treated less favorably based on their age"

Please give me an example scenario of where discrimination of age happens on Warzone. I don't feel like there are any, so why should we even include it in the rules, as if it's some kind of important thing? People will only be confused about this rule and interpret it wrongly.

Comment (residal) [reply src/pages/ServerRules.js:57]: Agreed, I'll go ahead and remove this as I no longer think it's necessary.

**Comment by jellz** [comment]
Why did you revert my changes of making the rules lines consistent with the “is not allowed” “is strictly prohibited” “is not tolerated” etc.? Unless there is a good reason I think the rules should have consistent endings with few exceptions.

**Comment by residal** [comment]
> Why did you revert my changes of making the rules lines consistent with the “is not allowed” “is strictly prohibited” “is not tolerated” etc.? Unless there is a good reason I think the rules should have consistent endings with few exceptions.

Sorry about that, my intention was not to directly remove **your** changes, as I did not realize it was apart of your edits. I had looked over the rules and thought that it needed a change to the wording and some more information.

Feel free to change back anything to keep up with consistency.

**Pull request merged**